test/1.4/lib/T_map_target_connect.py

```diff
@@ -53,13 +53,13 @@
 
 obj.x_address = 0
 obj.log_level = 2
-with stest.expect_log_mgr(obj, log_type="spec-viol"):
+with stest.expect_log_mgr(obj.bank.z, log_type="spec-viol"):
     rex = re.compile("^failed to read 8 bytes @ 0x0")
     with stest.expect_log_mgr(obj, log_type="info", regex=rex):
         _ = obj.x_value
 
     rex = re.compile("^failed to write 8 bytes @ 0x0")
-    with stest.expect_exception_mgr(simics.SimExc_Attribute):
-        with stest.expect_log_mgr(obj, log_type="info", regex=rex):
+    with stest.expect_log_mgr(obj, log_type="info", regex=rex):
+        with stest.expect_exception_mgr(simics.SimExc_Attribute):
```

Comment:
I took the liberty of correcting this. Before, the `stest.expect_log_mgr` didn't do what it was supposed to do due to the thrown exception.